src/here-xyz.ts

```diff
@@ -861,15 +751,7 @@ async function getCentreLatitudeOfSpace(spaceId: string, token: string | null =
 }
 
 async function getStatisticsData(spaceId: string, token: string | null = null) {
-    const response = await execute(
-        "/hub/spaces/" + spaceId + "/statistics",
-        "GET",
-        "application/json",
-        null,
-        token,
-        true
-    );
-    return response.body;
+    return xyzComm.getStatisticsData(spaceId, token)
```

Comment:
directly use xyzComm.getStatisticsData(..) in all the call rather than creating new method getStatisticsData(..) here

Reply:
Have imported getStatisticsData from ```xyzCommon.ts``` file